source/accessibility.html.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: core
+title: Accessibility statement
+review_in: 6 months
+hide_in_navigation: true
+last_reviewed_on: 2020-09-02
+---
+
+# Accessibility statement
+
+This accessibility statement applies to the GOV.UK developer documentation at [https://docs.publishing.service.gov.uk/](https://docs.publishing.service.gov.uk/).
+
+This website is run by the GOV.UK developer team at the Government Digital Service (GDS). We want as many people as possible to be able to use this website. For example, that means you should be able to:
+
+- change colours, contrast levels and fonts
+- zoom in up to 300% without the text spilling off the screen
+- navigate most of the website using just a keyboard
+- navigate most of the website using speech recognition software
+- listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)
+
+We’ve also made the website text as simple as possible to understand.
+
+[AbilityNet](https://mcmw.abilitynet.org.uk/) has advice on making your device easier to use if you have a disability.
+
+## How accessible this website is
+
+We know some parts of this website are not fully accessible.
+
+- Some links are broken.
+- Some tables are structured in an inaccessible way.
+- Some headings do not fit with the content that follows.
+- Some links do not clearly explain where they lead, or that they lead to external sites.
+- Some pages skip heading levels.
+- Some images do not have good alternative text, or have no alternative text.
+- Some content uses emojis without alternative text.
+- There are some links to documents in PDF format that are not accessible.
+
+## Feedback and contact information
+
+If you need any part of this service in a different format like large print, audio recording or braille, [contact us using the GOV.UK contact form](https://www.gov.uk/contact/govuk).
+
+We’ll consider your request and get back to you within 5 working days.
+
+## Reporting accessibility problems with this website
+
+We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, [contact us using the GOV.UK contact form](https://www.gov.uk/contact/govuk).
+
+### Enforcement procedure
+
+The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018
+(the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, [contact the Equality Advisory and Support Service (EASS)](https://www.equalityadvisoryservice.com/).
```

Comment:
This looks like an unexpected newline (all other paragraphs are encapsulated on their own line).

Reply:
Yep, good point, I've fixed that!